redirects/17.0.txt

```diff
@@ -57,4 +57,8 @@ applications/marketing/marketing_automation/advanced/understanding_metrics.rst a
 applications/marketing/marketing_automation/getting_started/first_campaign.rst applications/marketing/marketing_automation.rst                            # getting_started/first_campaign --> marketing_automation
 applications/marketing/marketing_automation/getting_started/target_audience.rst applications/marketing/marketing_automation/target_audience.rst           # getting_started/* --> marketing_automation/*
 applications/marketing/marketing_automation/getting_started/testing_running.rst applications/marketing/marketing_automation/testing_running.rst           # getting_started/* --> marketing_automation/*
-applications/marketing/marketing_automation/getting_started/workflow_activities.rst applications/marketing/marketing_automation/workflow_activities.rst   # getting_started/* --> marketing_automation/*
\ No newline at end of file
+applications/marketing/marketing_automation/getting_started/workflow_activities.rst applications/marketing/marketing_automation/workflow_activities.rst   # getting_started/* --> marketing_automation/*
+
+# applications/point_of_sale
+
+applications/sales/point_of_sale/pricing/digital_labels.rst applications/sales/point_of_sale/pricing/electronic_labels.rst                                # digital_labels --> electronic_labels
```

Comment:
Other redirection rules have "big categories" and not per app. I see that PoS had a dedicated section in older redirect rules, but we should keep it under "sales" IMO.

```suggestion
# applications/sales

applications/sales/point_of_sale/pricing/digital_labels.rst applications/sales/point_of_sale/pricing/electronic_labels.rst                                # digital_labels --> electronic_labels
```